utils/utility.py

```diff
@@ -0,0 +1,691 @@
+import numpy as np
```

Comment:
I cannot see where this file is used. Is there a file missing in the commit or this file not need?

Reply:
Thank you for your comments. 
You are right. `utils/utility.py` is not needed. I must have added it by mistake.
I have also renamed the `python` folder to `particle_net`, along with the necessary name changes.
I tested everything after the changes, and it works.